warning.i

```diff
@@ -3,4 +3,7 @@
     #include "Warning.h"
 %}
 
-%include Warning.h
\ No newline at end of file
+%include "std_list.i"
+%include Warning.h
+
+%template(WarningList) std::list<csm::Warning>;
```

Comment:
Adds support for a list of warnings.  A few of the csm methods pack lists of warnings.